docs/source/markdown/podman-image-trust.1.md.in

```diff
@@ -17,13 +17,20 @@ as an example, `docker` is the transport and `docker.io` is the registry host.
 Trust is defined in **/etc/containers/policy.json** and is enforced when a user attempts to pull
 a remote image from a registry.  The trust policy in policy.json describes a registry scope (registry and/or repository) for the trust.  This trust can use public keys for signed images.
 
-The scope of the trust is evaluated from most specific to the least specific. In other words, a policy may be defined for an entire registry.  Or it could be defined for a particular repository in that registry. Or it could be defined down to a specific signed image inside of the registry.
+The scope of the trust is evaluated from most specific to the least specific. In other words, a policy may be:
 
-For example, the following list includes valid scope values that could be used in policy.json from most specific to the least specific:
+ * Defined to an entire registry.
+ * Defined to a particular repository in that registry.
+ * Defined to a specific signed image inside of the registry.
+
+The following list are examples of valid scope values used in policy.json from most specific to the least specific:
 
 docker.io/library/busybox:notlatest
+
 docker.io/library/busybox
+
 docker.io/library
+
```

Comment:
Are the added blank line intentional here?

Reply:
Yes this makes the man page look much better, otherwise the list gets concatenated to a single line.